library/core/src/ffi/c_str.rs

```diff
@@ -169,24 +182,45 @@ impl fmt::Debug for CStr {
     }
 }
 
-#[stable(feature = "cstr_default", since = "1.10.0")]
-impl Default for &CStr {
-    fn default() -> Self {
-        const SLICE: &[c_char] = &[0];
-        // SAFETY: `SLICE` is indeed pointing to a valid nul-terminated string.
-        unsafe { CStr::from_ptr(SLICE.as_ptr()) }
+/// Converts a `&'static str` to a `&'static Cstr`.
```

Comment:
```suggestion
/// Converts a string literal to a `&'static Cstr`.
```

The macro uses `concat` inside so it likely won't accept e.g. the output of a function that returns `&'static str`.

Reply:
Thanks, I already thought about changing this after reading this again myself.